src/lib/dynamic-render/selectors.js

```diff
@@ -0,0 +1,35 @@
+import {
+    getIsFetchingWithId,
+    getIsLoadingWithId,
+    LoadingState
+} from '../../reducers/project-state';
+
+// Selectors here can provide a descriptive interface for when delay arguments
+// should be which values. If we use only these common functions we can use that
+// as a way to shortcut all of the delays gates. By replacing the functions on
+// Delay with ones that return true.
```

Comment:
Hmmmm? I think there might be some punctuation out of place or something, but these sentences kinda frazzled my brain.

Reply:
Ah. Erm. Probably more than punctuation. Hmm.